pkg/kubelet/dockertools/manager.go

```diff
@@ -47,6 +47,7 @@ import (
 	"k8s.io/kubernetes/pkg/kubelet/prober"
 	"k8s.io/kubernetes/pkg/kubelet/qos"
 	kubeletTypes "k8s.io/kubernetes/pkg/kubelet/types"
+	kubetypes "k8s.io/kubernetes/pkg/kubelet/types"
```

Comment:
"kubetypes" is used in master branch.
I can remove the added import.

The other issue I faced while creating the cherry-pick was that, on master branch, we now have HostNetwork field within SecurityContext within Spec within Pod type.

But the release branch does not have SecurityContext in Spec.
HostNetwork field is present in the Spec as a top level field.

So, I had to make custom changes around that.

Let me know your thoughts.
I assumed that we don't want to pull in that Spec change.